src/EnergyPlus/OutputProcessor.cc

```diff
@@ -6182,8 +6182,9 @@ GenOutputVariablesAuditReport()
 			OpaqSurfWarned = true;
 		}
 		if ( ! Rept ) {
-			ShowWarningError( "The following Report Variables were requested but not generated" );
-			ShowContinueError( "because IDF did not contain these elements or misspelled variable name -- check .rdd file" );
+			ShowWarningError( "The following Report Variables were requested but not generated -- check.rdd file" );
+			ShowContinueError( "Either the IDF did not contain these elements, the variable name is misspelled," );
+			ShowContinueError( "or the requested variable is an advanced output which requires Output : Diagnostics, DisplayAdvancedReportVariables;" );
```

Comment:
@rraustad The intent is to provide guidance when a user requests such output variables without using Diagnostics, DisplayAdvancedReportVariables. This is what Mike suggested. Please instruction in the issue.